server/sitemap.ts

```diff
@@ -25,9 +26,23 @@ const generateSitemapPage = (
   root: string,
   { loc, lastmod = defaultLastmod, changefreq = "daily", priority }: SitemapPage
 ) => {
+  const firstLvlNav = findFirstLvlNav(loc);
+  let updateLoc = loc;
```

Comment:
Maybe something a bit more explicit like...
```suggestion
  let canonicalLoc = loc;
```